lib/plenario2/schemas/meta.ex

```diff
@@ -61,10 +61,11 @@ defmodule Plenario2.Schemas.Meta do
   ##
   # schema functions
 
+  # TODO(heyzoos) inconsistent naming
```

Comment:
Does this belong in actions, or should the functions you added in actions to get field names and constraints belong in here?

Reply:
Also, what's inconsistent about this name?

Reply:
Just something that was tripping me up. Some things are written as `data_set` while others are `dataset`. It's about 50/50 with the functions so I wasn't sure which way to go. I left the note as a reminder.